README.md

```diff
@@ -176,3 +179,47 @@ hocr-split file.html pattern
 
 Split a multipage hOCR file into hOCR files containing one page each.
 The pattern should something like "base-%03d.html"
+
+## Unit tests
+
+The unit tests are written using the [tsht](https://github.com/kba/tsht) framework.
+
+### Running the full test suite:
+
+```sh
+./test/tsht
+```
+
+### Running a single test
+
+```sh
+./test/tsht <path-to/unit-test.tsht>
+```
+
+e.g.
+
+```sh
+./test/tsht hocr-pdf/test-hocr-pdf.tsht
+```
+
+### Writing a test
+
+Please see the documentation in the [tsht](https://github.com/kba/tsht) repository and
+take a look at the existing [unit tests](./test/).
+
+1) Create a new directory under `./test`
+2) Copy any test assets (images, hocr files...) to this directory
+3) Create a file `<name-of-your-test>.tsht` starting from this template:
+
+```sh
+#!/usr/bin/env tsht
+
+# adjust to the number of your tests
+plan 1
```

Comment:
This is not always given in the tests below. I would replace the comment with `# number of test statements in the following` or something like that. Is it important to add this line?